preload_database.sql

```diff
@@ -3223,30 +3223,30 @@ INSERT INTO "parameter" VALUES(2708,'ph_light_measurements',3,3,NULL,65,5,'Array
-INSERT INTO "parameter" VALUES(2732,'temperature20',9,1,NULL,90,7,'Seawater Temperature in Spatial Grid 20',NULL,4,NULL,NULL,NULL,'Seawater Temperature in Spatial Grid 20 is temperature measured at 1 of 24 locations along the thermistor string that is positioned in a spatial array. The position of each thermistor is given in the metadata, using <x y z> coordinates in centimeters with an origin at the seafloor beneath thermistor #2. Units: degrees Celsius',NULL,NULL);
-INSERT INTO "parameter" VALUES(2733,'temperature21',9,1,NULL,90,7,'Seawater Temperature in Spatial Grid 21',NULL,4,NULL,NULL,NULL,'Seawater Temperature in Spatial Grid 21 is temperature measured at 1 of 24 locations along the thermistor string that is positioned in a spatial array. The position of each thermistor is given in the metadata, using <x y z> coordinates in centimeters with an origin at the seafloor beneath thermistor #2. Units: degrees Celsius',NULL,NULL);
-INSERT INTO "parameter" VALUES(2734,'temperature22',9,1,NULL,90,7,'Seawater Temperature in Spatial Grid 22',NULL,4,NULL,NULL,NULL,'Seawater Temperature in Spatial Grid 22 is temperature measured at 1 of 24 locations along the thermistor string that is positioned in a spatial array. The position of each thermistor is given in the metadata, using <x y z> coordinates in centimeters with an origin at the seafloor beneath thermistor #2. Units: degrees Celsius',NULL,NULL);
-INSERT INTO "parameter" VALUES(2735,'temperature23',9,1,NULL,90,7,'Seawater Temperature in Spatial Grid 23',NULL,4,NULL,NULL,NULL,'Seawater Temperature in Spatial Grid 23 is temperature measured at 1 of 24 locations along the thermistor string that is positioned in a spatial array. The position of each thermistor is given in the metadata, using <x y z> coordinates in centimeters with an origin at the seafloor beneath thermistor #2. Units: degrees Celsius',NULL,NULL);
-INSERT INTO "parameter" VALUES(2736,'temperature24',9,1,NULL,90,7,'Seawater Temperature in Spatial Grid 24',NULL,4,NULL,NULL,NULL,'Seawater Temperature in Spatial Grid 24 is temperature measured at 1 of 24 locations along the thermistor string that is positioned in a spatial array. The position of each thermistor is given in the metadata, using <x y z> coordinates in centimeters with an origin at the seafloor beneath thermistor #2. Units: degrees Celsius',NULL,NULL);
+INSERT INTO "parameter" VALUES(2713,'temperature01',9,1,NULL,90,7,'Seawater Temperature in Spatial Grid 01',NULL,4,NULL,NULL,NULL,'Seawater Temperature in Spatial Grid 01 is temperature measured at 1 of 24 locations along the thermistor string that is positioned in a spatial array. The position of each thermistor is given in the metadata, using <x y z> coordinates in centimeters with an origin at the seafloor beneath thermistor #2. Units: degrees Celsius',3,NULL);
```

Comment:
why did these change from `NULL` to `3`?

Reply:
Looks like someone changed something somewhere but I can't see where it's changed. it change NULL to 3 , without any of my changes when i ran ./load_preload.py